build.gradle

```diff
@@ -0,0 +1,47 @@
+import java.util.regex.Pattern
+
+/** Defines a custom task for updating version numbers. */
+abstract class UpdateVersionTask extends DefaultTask {
+
+    static final def VERSION_REGEX = "([0-9]+)\\.([0-9]+)\\.([0-9]+)(-SNAPSHOT)?"
```

Comment:
Add `^version=` to make sure this doesn't pick up any other lines.

Reply:
We're parsing into a `java.util.Properties`, so there's no need to match the raw text here.

Reply:
Oh, of course! Sorry, didn't think of that.